therapy/main.py

```diff
@@ -99,30 +111,48 @@ def search(q: str = Query(..., description=q_descr),
     return response
 
 
-merged_matches_summary = "Given query, provide merged normalized record."
-merged_response_descr = "A response to a validly-formed query."
-merged_q_descr = "Therapy to normalize."
-
-
 @app.get("/therapy/normalize",
          summary=merged_matches_summary,
-         operation_id="getQuerymergedResponse",
+         operation_id="getMergedRecord",
          response_description=merged_response_descr,
          response_model=NormalizationService,
          description=normalize_description)
-def normalize(q: str = Query(..., description=merged_q_descr),
+def normalize(q: str = Query(..., description=normalize_q_descr),
               infer_namespace: bool = Query(True, description=infer_descr)
               ) -> NormalizationService:
     """Return merged strongest-match concept for query string provided by
     user.
 
     :param q: therapy search term
     :param bool infer_namespace: if True, try to infer namespace from query term.
-    :returns: JSON response with matching normalized record and source metadata
+    :returns: JSON response with matching normalized record provided as a
+    TherapyDescriptor, and source metadata
     """
     try:
         response = query_handler.normalize(html.unescape(q),
                                            infer_namespace)  # type: ignore
     except InvalidParameterException as e:
         raise HTTPException(status_code=422, detail=str(e))
     return response
+
+
+@app.get("/therapy/normalize_unmerged",
+         summary=unmerged_matches_summary,
+         operation_id="getUnmergedRecords",
+         response_description=unmerged_response_descr,
+         response_model=UnmergedNormalizationService,
+         description=unmerged_normalize_description)
+def normalize_unmerged(q: str = Query(..., description=normalize_q_descr),
+                       infer_namespace: bool = Query(True, description=infer_descr),
+                       ) -> UnmergedNormalizationService:
+    """Return all individual records associated with a normalized concept.
+
+    :param q: therapy search term
+    :param bool infer_namespace: if True, try to infer namespace from query term.
+    :returns: JSON response with matching normalized record and source metadata
+    """
+    try:
+        response = query_handler.normalize_unmerged(html.unescape(q), infer_namespace)
```

Comment:
I think we should be using `unquote` (`from urllib.parse import unquote`) rather than `html.unescape`
```suggestion
        response = query_handler.normalize_unmerged(unquote(q), infer_namespace)
```

Reply:
👍 👍 yeah I encountered a few issues related to this and am gonna do a big fix in #261